### ISSUE TITLE
Import Classes not working

### ISSUE THREAD

**Issue by nicolo-crimaldi** [ExalFabu/tptp#47]
TL;DR When I try to import classes from Unipa with any enrollment year I can only find the ones for the current academic year and in english.

#  Wrong academic year
When I try to import classes the study plan is always at the current academic year even though I specify my actual enrollment year. I tried to change the values but nothing changes. I think I'm going to try with other courses to see if that's the problem (_Maybe I should have checked before opening the issue but I dont' think so_ :wink: ).
## Possible fix
I suspect something's wrong with the search. I mean that when I put the enrollment year I can only type the solar year but when I followed the post request for the search to the UNIPA site I noticed that in the field for academic year of the educational offer I have to insert an academic year In the format of _AAAA/AAAA_ (Actually I have not checked the code for this yet)

# Only works in english
I tried to search for courses in italian but it doesn't find anything even with different academic years.
